### DISCUSSION TITLE
feature req - no sharing of database connections

### DISCUSSION THREAD

**Discussion by tooptoop4** [apache/superset#22263] [Ideas]
After enabling LDAP authentication on superset UI login I would like user1 to only be able to see/use database (ie a connection to redshift with user1 credentials) connections created by user1. Likewise, I would like user2 to only be able to see/use database (ie a connection to redshift with user2 credentials) connections created by user2. so on and so forth for user 3,4,5...n

**Comment by rusackas** [comment]
This isn't a typical flow, but I think I understand your desire. Essentially, you're sharing a Superset instance, but not sharing any of the database connections. This means there would be no collaboration with shared datasets, dashboards, charts, alerts/reports, etc. On the one hand, this vastly reduces the collaborative intent of Superset, but on the other hand, I can see some valid use cases to let people work independently with private information. 

Do you think you would want people to be able to override this, and enable other individuals to share databases (or datasets) with other users, or do you want this to be some sort of permanent-lock where access would be completely barred to others (e.g. a "private" checkbox in the datasource modal)?

**Reply by tooptoop4** [reply]
permanent lock

**Comment by rusackas** [comment]
Since this is a feature request, I'm going to go ahead and move it to Discussions, and we can take it up there. Thanks again!
